doc/tutorial/tutorial.tex

```diff
@@ -3581,6 +3582,91 @@ \subsection*{Notes}
+        \item{Directory: {\tt examples/example31/}}
+        \item{Input files}
+        \begin{itemize}
+                \item{ {\tt W.scf} {\it The \pwscf\ input file for ground state
+                                calculation}}
+                \item{ {\tt W.nscf}  {\it The \pwscf\ input file to obtain Bloch
+                                states on a uniform grid}}
+                \item{ {\tt W.pw2wan}  {\it The input file for {\tt pw2wannier90}}}
+                \item{ {\tt W.proj}  {\it The input file for {\tt projwfc}}}
+                \item{ {\tt generate\_weights.sh} {\it The bash script to extract the projectabilities from the output of {\tt projwfc} }}
+                \item{ {\tt W.win}  {\it The {\tt wannier90} input file}}
+        \end{itemize}
+\end{itemize}
+
+\begin{enumerate}
+	\item Run \pwscf\ to obtain the ground state of tungsten\\
+	{\tt pw.x -in W.scf > scf.out}
+	
+	\item Run \pwscf\ to obtain the Bloch states on a $10\times10\times10$ uniform $k$-point
+	grid\\ 
+	{\tt pw.x -in W.nscf > nscf.out}
+	
+	\item Run \wannier\ to generate a list of the required overlaps (written
+	into the {\tt W.nnkp} file)\\
+	{\tt wannier90.x -pp W}
+	
+        \item Run {\tt projwfc} to compute the projectabilities of the Bloch states
+              onto the Bloch sums obtained from the PAOs in the pseudopotential \\
+        {\tt projwfc.x -in W.proj > proj.out}
+
+        \item Run {\tt generate\_weights} to extract the projectabilitites from {\tt proj.out}
+              in a format suitable to be read by Xmgrace or gnuplot \\
+	{\tt ./generate\_weights.sh}
+
+        \item Plot the projectabilities and fit the data with the complementary error function
+              $$f(\epsilon;\mu,\sigma) = \frac{1}{2}\mathrm{erfc}(-\frac{\mu - \epsilon}{\sigma}).$$
+		We are going Xmgrace to plot the projectabilities and perform the fitting. Open Xmgrace \\
+	{\tt xmgrace }
+		
+	To Import the {\tt p\_vs\_e.dat} file, click on {\tt Data} from the top bar and then {\tt Import -> ASCII...}.
+	At this point a new window {\tt Grace: Read sets} should pop up . Select {\tt p\_vs\_e.dat} in the {\tt Files} section, click {\tt Ok} at the bottom and close the window. You should now be able to see a quite noisy function that is bounded between 1 and 0. You can modify the appearence of the plot by clicking on {\tt Plot} in the top bar and then {\tt Set appearance...}. In the {\tt Main} section of the pop-up window change the symbol type from {\tt None} to {\tt Circle} and symbol size from 100 to 25. Also change symbol color from black to blue. Change the line type from straight to none. Move to the {\tt Symbols} section and change the filling pattern from none to full (indentified by a full black square) and then close the window. For the fitting, go to {\tt Data -> Transformations -> Non-linear curve fitting}. In this window, select the source from the {\tt Set} box and in the {\tt Formula} box insert the following \\
```

Comment:
Remove space after `pop up` before the dot.
Also, it is really needed to ask to use 'fill' as a pattern and to make dots blue? I would remove this as it just makes the description longer and it's not critical. I don't think it's a problem if the plot is slightly different from the figure.

Reply:
Yes I think you are right. I'll remove the section where I describe how to modify the graph appareance.